src/main/scad/akaki-kuumeri/akakis-picnic/akakis-picnic.yaml

```diff
@@ -0,0 +1,21 @@
+thing-id: 4185289
+name: "Akaki's Picnic -- A packing puzzle series by Akaki Kuumeri"
+tags: [puzzle, 3D_puzzle]
+images: [chocolate.jpg]
+targets: [basket, chocolate, wine, vegetable, fruit, subway, egg, ice_cream, cake, coffee, sandwich, salmiakki, chicken, hamburger]
```

Comment:
Note: You can add your Fusion-generated STL to the akakis-picnic directory, and put in the yaml file

aux_files: [basket.stl]

Then the PPP scripts will automatically include basket.stl in the Thingiverse listing.